cli/upload/upload.go

```diff
@@ -53,7 +54,8 @@ func NewCommand() *cobra.Command {
 
 	uploadCommand.Flags().StringVarP(&fqbn, "fqbn", "b", "", "Fully Qualified Board Name, e.g.: arduino:avr:uno")
 	uploadCommand.Flags().StringVarP(&port, "port", "p", "", "Upload port, e.g.: COM10 or /dev/ttyACM0")
-	uploadCommand.Flags().StringVarP(&importDir, "input-dir", "", "", "Direcory containing binaries to upload.")
+	uploadCommand.Flags().StringVarP(&importDir, "input-dir", "", "", "Directory containing binaries to upload.")
+	uploadCommand.Flags().StringVarP(&importFile, "input-file", "", "", "Binary file to upload.")
```

Comment:
We should re-do the old shortcut `-i` too `--input-file/-i`

Reply:
See e10d0ff